packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -191,8 +199,15 @@ function ExploreTemplates() {
       setActiveTemplate(newTemplate);
       setActiveTemplateIndex(offset);
       replace(`?id=${newTemplate.id}&isLocal=${newTemplate.isLocal}`);
+      speak(
+        sprintf(
+          /* translators: %s: template title */
+          __('Viewing %s', 'web-stories'),
```

Comment:
💭 Haven't found a way to read the template description using voice box. Maybe it should be announced when the template is opened?

Reply:
This doesn't seem to be reading the title for me when I switch to a new template. Does it for y'all?